airflow-core/src/airflow/ui/src/i18n/locales/de/common.json

```
@@ -0,0 +1,78 @@
+{
+  "defaultToGraphView": "Graph-Ansicht als Standard",
+  "defaultToGridView": "Gitter-Ansicht als Standard",
+  "logout": "Abmelden",
+  "switchToDarkMode": "Zum Dunkelmodus wechseln",
+  "switchToLightMode": "Zum Hellmodus wechseln",
+  "timezone": "Zeitzone",
+  "user": "Benutzer",
+  "language": {
+    "chinese": "Traditionelles Chinesisch",
+    "english": "Englisch",
+    "german": "Deutsch",
+    "select": "Sprache wählen"
+  },
+  "nav": {
+    "home": "Start",
+    "assets": "Datenbestände",
+    "browse": "Browsen",
+    "admin": "Verwaltung",
+    "docs": "Dokumentation",
+    "plugins": "Plug-ins"
+  },
+  "browse": {
+    "auditLog": "Prüf-Log",
+    "xcoms": "Datenwerte"
+  },
+  "admin": {
+    "Variables": "Variablen",
+    "Pools": "Pools",
+    "Providers": "Provider",
```

(All plural in the menue)
```suggestion
    "Providers": "Providers",
```